src/lib/special_floors.lua

```diff
@@ -8,12 +8,15 @@ ROOMS_BEFORE_SKIP = 4
 
 def configure_floor(
     name: str,
-    desc: object, # str or lambda that returns string
+    str_desc: str = None,
+    lambda_desc: function = lambda dungeon_data: "Example", # optional lambda that returns a string value

```

Comment:
```suggestion
    lambda_desc: None
```

In the event we don't have str_desc or lambda_desc let's just throw an error